renovate/action.yml

```diff
@@ -0,0 +1,138 @@
+name: 'Renovate with Dynamic Secrets'
+description: 'Run Renovate with auto-detected RENOVATE_* secrets from GitHub Secrets'
+author: 'EIDP'
+
+branding:
+  icon: 'refresh-cw'
+  color: 'blue'
+
+inputs:
+  github-app-id:
+    description: 'The ID of the GitHub App used for Renovate'
+    required: true
+  
+  github-app-pem:
+    description: 'The private key (in PEM format) of the GitHub App used for Renovate'
+    required: true
+  
+  all-secrets:
```

Comment:
Should there also be an `all-variables` for non-sensitive values that Renovate takes, e.g. `RENOVATE_DOCKER_USERNAME`?